ext/json/json_encoder.c

```diff
@@ -61,9 +61,14 @@ static inline void php_json_pretty_print_indent(smart_str *buf, int options, php
 {
 	int i;
 
+	int indent_length = encoder->indent_str ? ZSTR_LEN(encoder->indent_str) : 0;
```

Comment:
```suggestion
	size_t indent_length = encoder->indent_str ? ZSTR_LEN(encoder->indent_str) : 0;
```
And you probably want to change the definition of `i` to also be a `size_t`